Quantum-Control-Applications/Superconducting/Single Fixed Transmon/rb_without_singleshot_readout.py

```diff
@@ -13,7 +13,7 @@
 inv_gates = [int(np.where(c1_table[i, :] == 0)[0][0]) for i in range(24)]
 max_circuit_depth = int(3 * qubit_T1 / x180_len)
 delta_depth = 1
-num_of_sequences = 50
+num_of_sequences = 5
```

Comment:
why did you lower this?

Reply:
Just for a quick test and forgot to put it back